src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -97,14 +97,18 @@ public Token login(String username, String password)
      * @return authentication token as {@link Token}
      * @throws SaltStackException if anything goes wrong
      */
-    public Token login(String username, String password, String eauth)
+    public Token login(final String username, final String password, final String eauth)
             throws SaltStackException {
-        JsonObject json = new JsonObject();
-        json.addProperty("username", username);
-        json.addProperty("password", password);
-        json.addProperty("eauth", eauth);
+        Map<String, String> kwargs = new LinkedHashMap() {
```

Comment:
I am fine with that, just wrap lines before reaching 92 characters and indent correctly. We might want to finalize the parameters of the (now merged) `*Async` methods in a separate commit or I will do it after we merged this.